src/core/services/ripple/ripple.js

```diff
@@ -252,32 +265,57 @@ InkRippleCtrl.prototype.bindEvents = function () {
  */
 InkRippleCtrl.prototype.handleMousedown = function (event) {
   if ( this.mousedown ) return;
+  this.mousedown = true;
+  //Return and wait until mouseup if this is an explode ripple
+  if (this.explode) return;
+  createRippleFromEvent(this,event);
+};
+
+/**
+ * If this instance is an explode create the ripple, otherwise,
+ * Either remove or unlock any remaining ripples when the user mouses off of the element (either by
+ * mouseup, touchend or mouseleave event)
+ */
+InkRippleCtrl.prototype.handleMouseup = function (event) {
+  //If this is an explode ripple and it is not a right click create the ripple
+  if (this.explode && event.which !== 3){
```

Comment:
Space brace here as well. And about the `3` here, IMO we should provide this in the `$mdConstant` service. 

https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/which